src/components/Player/Player.js

```diff
@@ -22,7 +57,7 @@ class Player extends Component {
 
             <br />
 
-            <span>Track 1</span>
+            <span>{this.model.track ? this.model.track.title : ''}</span>
```

Comment:
I think you can also move it to getter. Logic/calculations in render should be avoided.